src/core/thread/osthread.d

```diff
@@ -1978,6 +1979,16 @@ extern (C) void thread_init() @nogc
     }
     else version (Posix)
     {
+        if ( suspendSignalNumber == 0 )
```

Comment:
Use or avoid the other condition completly since `thread_setGCSignals` guarantees to see both variables:
```suggestion
        if ( suspendSignalNumber == 0 || resumeSignalNumber == 0 )
```

And reuse `SIGRTMIN` value since it might be a function call, see https://github.com/dlang/druntime/blob/a17bb23b418405e1ce8e4a317651039758013f39/src/core/sys/posix/signal.d#L138 .